docs/content/docs/build/containers.md

```diff
@@ -183,38 +166,54 @@ A container is considered dirty in the following cases:
   In some cases this can be due to lack of network connection.
   If the network connection is down, it needs to be restored for the pending changes to be acknowledged.
 
-In terms of user experience, keep in mind that the `isDirty` flag will flicker as pending changes are awaiting acknowledgement.
-Therefore, the host may choose to incorporate some delay before interpreting the flag again, when basing the user experience off of its state.
-
 ```typescript {linenos=inline}
 if(container.isDirty) {
   container.on("saved", () => {
-    // safe to close the container
+    // safe to dispose or disconnect the container
   });
 }
 ```
 
-### saved
+In terms of user experience, keep in mind that the `isDirty` flag will flicker as pending changes are awaiting acknowledgement.
+Therefore, the host may choose to incorporate some delay before interpreting the flag again, when basing the user experience off of its state.
 
 The container emits the *saved* event to notify the caller that all the local changes have been acknowledged by the service and the document is marked as clean.
 
 ```typescript {linenos=inline}
 container.on("saved", () => {
-    // allow the user to edit
+    // all pending edits have been saved at this point.
 });
 ```
 
-### dirty
-
 The container emits the *dirty* event to notify the caller that there are local changes that have not been acknowledged by the service yet and the document is still in a dirty state.
 
 ```typescript {linenos=inline}
 container.on("dirty", () => {
-    // prevent the user from editing to avoid data loss
+    // container has pending changes that need to be ack-ed by the service.
 });
 ```
 
-### Initial objects
+### Disposing a container
+
+The container object may be disposed to remove any server connections and clean up registered events. Once a container is disposed, your code must call `getContainer` again if it needs to  be reloaded.
+
+```typescript {linenos=inline}
+container.dispose();
+
+const disposed = container.disposed;
+
+container.on("disposed", () => {
+    // handle event cleanup to prevent memory leaks
+});
+```
+
+As mentioned above, you probably want to make sure all pending changes are saved prior to calling `dispose`.
+
+### Deleting a container
+
+Deleting a container is a service-specific feature, so you should refer to the documentation associated with the specific Fluid service you are using. See [Available Fluid Services]({{< relref "service-options.md" >}}) for more information about Fluid service options.
```

Comment:
Why is this "service-specific" feature? We should elaborate this further. If we are implying we don't want to expose this feature through API we should elaborate on why-s. The way we phrase it here it sounds like there is API we could use, but its service specific, and user should go look for it. Then when you look at our client packages, none of them offer this capability. The intention may be to communicate something different here? AFR has this capability: https://docs.microsoft.com/en-us/azure/azure-fluid-relay/how-tos/container-deletion.

Reply:
It's service-specific because we think that data management of that type belongs at the service level. For Azure, for example, there are lots of existing storage APIs that AFR is integrating with for managing the lifecycle of the container data (when you think of containers as documents/files). Azure customers expect to be able to manage their "Fluid data" in the same way as their other data.

Reply:
I had few questions on this topic, but we can separate them from this PR. This is good clarifications for what I needed here.